enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -139,6 +133,20 @@ public OnboardingStartResponse startOnboarding(OnboardingStartRequest request) t
             process.setTimestampCreated(new Date());
         }
         process = onboardingProcessRepository.save(process);
+
+        // Check for brute force attacks
+        Calendar c = GregorianCalendar.getInstance();
+        c.add(Calendar.HOUR, -24);
+        Date timestampCheckStart = c.getTime();
+        int existingProcessCount = onboardingProcessRepository.countProcessesAfterTimestamp(userId, timestampCheckStart);
+        if (existingProcessCount > onboardingConfig.getMaxProcessCountPerDay()) {
+            process.setStatus(OnboardingStatus.FAILED);
+            process.setErrorDetail(ERROR_TOO_MANY_PROCESSES_PER_USER);
+            onboardingProcessRepository.save(process);
+            logger.warn("Maximum number of processes per day reached for user: {}", userId);
```

Comment:
Maybe we can include also the violated `maxProcessCountPerDay` value in the log message?

Reply:
Yes, I will add it.